tests.js

```diff
@@ -0,0 +1,88 @@
+require('required_env')([ 'AUTH_URL' ]);
+import Driver from './src';
+import test from 'blue-tape';
+import bandname from 'bandname';
+
+// Set to your auth api key
+const TEST_PASSWORD = 'hai';
```

Comment:
This was previously just part of the AUTH_URL environment variable. The form was:

`AUTH_URL="http://wut:hai@localhost:3000"`

Is there a reason to split it out? If so, and it's compelling, we should expose TEST_PASSWORD as an env variable too. I suspect we should continue just taking it in as a single URL, though.